azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ChatCompositeActivity.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.presentation;
+
+import android.content.Context;
+import android.content.Intent;
+
+import com.azure.android.communication.ui.chat.ChatComposite;
+
+class ChatCompositeActivity {
```

Comment:
Is named Activity, but is not Activity. Do we need Impl?

Reply:
Maybe we should move into the Demo itself and not library (if not exposing activity mode for now)

Reply:
That is a name we can use for now, until it will become public. Then we will have a proper conversation on this name.

Reply:
sorry, forgot I was looking at Java and that it's package private.